metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/HiveConnectorFastPartitionService.java

```diff
@@ -65,6 +65,8 @@
  */
 @Slf4j
 public class HiveConnectorFastPartitionService extends HiveConnectorPartitionService {
+    private static final String SORT_BY_DATE_CREATED = "dateCreated";
+
```

Comment:
Is this constant used anywhere else? If so, you can put into our constant class.

Reply:
Just in the tests. Looks like there's one defined in DirectSqlGetPartition already. Updated to use that.